crates/accelerate/src/target_transpiler/mod.rs

```diff
@@ -547,19 +550,19 @@ impl Target {
     ///     bool: Returns ``True`` if the instruction is supported and ``False`` if it isn't.
     #[pyo3(
         name = "instruction_supported",
-        signature = (operation_name=None, qargs=None, operation_class=None, parameters=None)
+        signature = (operation_name=None, qargs=TargetQargs::Global, operation_class=None, parameters=None)
```

Comment:
Do you think thiw syntax is valid here, it feels strange that qargs is global by default here? The python counterpart did have `None` as a default for everything, but for this we might prefer to use `Option<Qargs>` but I'm not quite sure.